src/app/common/s/service/traveler-auth-service.service.ts

```diff
@@ -38,4 +46,10 @@ export class TravelerAuthService {
     return moment(expiresAt);
```

Comment:
isLoggedIn() should return a boolean based on the function name. This is more like getExpirationToken()

Reply:
changed to return boolean